test/simple_integer_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class TestSimpleInteger < MiniTest::Unit::TestCase
+  CLASS = ::Semian::Simple::Integer
+
+  def setup
+    @integer = CLASS.new
+    @integer.value = 0
```

Comment:
This should not be necessary since you already call `#reset` in the initializer. It's worth writing a test for this.